Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -63,7 +63,7 @@ extension KeyboardShortcuts {
 
 			super.init(frame: .zero)
 			self.delegate = self
-			self.placeholderString = "Record Shortcut"
+            self.placeholderString = "Record Shortcut".localized
```

Comment:
Tab-indentation

Reply:
![image](https://user-images.githubusercontent.com/33802186/91556059-ac017700-e964-11ea-8189-1598fb562faf.png)
This looks great on my Xcode, that is strange.

Reply:
Install https://github.com/sindresorhus/refined-github and look at the diff and you'll see the difference in indentation. You could also turn on showing invisible in Xcode preferences.